FoundationsMastery/CustomString.cs

```diff
@@ -0,0 +1,75 @@
+﻿using FoundationsMastery.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FoundationsMastery
+{
+    public class CustomString : ICustomString
+    {
+        public CustomString()
+        {
+        }
+
+        public CustomString(IEnumerable<char> contents)
+        {
+            this.Contents = contents;
+        }
+
+        public IEnumerable<char> Contents { get; set; } 
+       
+        public int Length {
+            get
+            {
+                return this.Contents.Count(); 
+            }
+        } 
+
+        public void Clear() 
+        {
+            Contents = new List<char>(); 
+            Contents.ToList().Clear(); 
```

Comment:
By default, a new List _is_ empty, so the explicit `Clear` isn't technically necessary